WcaOnRails/app/views/competitions/_registration_requirements.html.erb

```diff
@@ -0,0 +1,59 @@
+<% if @competition.use_wca_registration? %>
+  <%= t('competitions.competition_info.create_wca_account_html', here: link_to(t('common.here'), new_user_registration_path)) %>
+  <br>
+  <%= t('competitions.competition_info.claim_wca_id_html', here: link_to(t('common.here'), profile_claim_wca_id_path)) %>
+  <br>
+  <%= t('competitions.competition_info.register_html', here: link_to(t('common.here'), competition_register_path(@competition))) %>
+  <br>
+<% else %>
+  <%= t('competitions.competition_info.register_html', here: link_to(t('common.here'), @competition.website, target: "_blank")) %>
+  <br>
+<% end %>
+<% if @competition.competitor_limit_enabled? %>
+  <%= t('competitions.competition_info.competitor_limit', competitor_limit: @competition.competitor_limit) %>
+  <br>
+<% else %>
+  <%= t('competitions.competition_info.no_competitor_limit') %>
+  <br>
+<% end %>
+<% if @competition.has_fees? %>
+  <%= t('competitions.competition_info.entry_fee', base_entry_fee: format_money(@competition.base_entry_fee)) %>
+  <br>
+  <% @competition.competition_events.each do |competition_event| %>
+    <% if competition_event.has_fee? %>
+      <dt><%= competition_event.event.name %></dt>
+      <dd><%= format_money competition_event.fee %></dd>
+    <% end %>
+  <% end %>
+<% else %>
+  <%= t('competitions.competition_info.no_entry_fee') %>
+  <br>
+<% end %>
+<% if @competition.refund_policy_percentage? %>
+  <%= t('competitions.competition_info.refund_policy_html', refund_policy_percentage: number_to_percentage(@competition.refund_policy_percentage, precision: 0), limit_date_and_time: wca_local_time(@competition.refund_policy_limit_date)) %>
```

Comment:
Have you checked what this will do if `@competition.refund_policy_limit_date` is NULL (as it will be for past competitions when we run the migration)? I suspect that `wca_local_time` will blow up...

Reply:
As `refund_policy_percent` will be 0, `wca_local_time` will never be called for past competitions.